ht/cogs/roll.py

```diff
@@ -45,6 +49,60 @@ async def armiger(self, ctx, user: converters.Armiger = None):
 		await self.add_rolls(embed, "AND NOT personal", user, "Artist gallery")
 		await ctx.send(embed = embed)
 
+	@commands.command(
+		help = "Looks up the symbology of a user's coat of arms.\nUses GreiiEquites' https://roll-of-arms.com as a source.",
+		aliases = ("s", "symbols")
+	)
+	async def symbolism(self, ctx, user: converters.Armiger = None):
+		if not user:
+			user = await ctx.bot.dbc.execute_fetchone(
+				"SELECT * FROM armigers_e WHERE discord_id == ?;", (ctx.author.id,)
+			)
+
+			if not user:
+				await ctx.typing()
+				await self.bot.get_cog("Bot tasks").sync_book()
+
+				user = await ctx.bot.dbc.execute_fetchone(
+					"SELECT * FROM armigers_e WHERE discord_id == ?;", (ctx.author.id,)
+				)
+
+			if not user: raise utils.CustomCommandError(
+				"Invalid armiger",
+				"There are no arms associated with your user account. "
+				"To find those of another user, follow the command with their username."
+				"If you wish to register your arms, follow the instructions at the Roll of Arms server."
+			)
+
+			async with aiohttp.ClientSession() as session:
+				async with session.get(f'https://roll-of-arms.com/wiki/GreiiN:{user[0]}') as response:
+					if response.status == 404:
+						utils.CustomCommandError(
+							"Armiger is not on roll-of-arms.com",
+							"The arms of the armiger are not on the https://roll-of-arms.com "
+							"website. If you would like to add your arms and related symbolism "
+							"to the website, please fill out the form pinned in the "
+                            "#announcements channel of the Roll of Arms server."
+						)
```

Comment:
```suggestion
		async with self.bot.session.get(f'https://roll-of-arms.com/wiki/GreiiN:{user[0]}') as response:
			if response.status == 404:
				raise utils.CustomCommandError(
					"Armiger is not on roll-of-arms.com",
					"The arms of the armiger are not on the https://roll-of-arms.com "
					"website. If you would like to add your arms and related symbolism "
					"to the website, please fill out the form pinned in the "
                    "#announcements channel of the Roll of Arms server."
				)
```

Each `aiohttp` session incurs networking overhead when starting it and shutting it down. I did this at first too, but now the bot has one global session saved to make things more efficient.

The code is also indented one level too far. This means it will fail if the user is found by the converter (i.e. if the first `if not user` condition is not met)

Also, the code style is to use double quotes everywhere.